prawns/identities/package.json

```diff
@@ -5,12 +5,10 @@
   "private": true,
   "scripts": {
     "db:init": "docker-compose -f ../../docker-compose.dev.yml down && docker-compose -f ../../docker-compose.dev.yml up -d && yarn build && yarn migrate",
-    "dev:khala": "node -r dotenv/config lib/processors/khalaProcessor.js",
-    "dev:polkadot": "node -r dotenv/config lib/processors/polkadotProcessor.js",
-    "dev:kusama": "node -r dotenv/config lib/processors/kusamaProcessor.js",
-    "process:khala": "node lib/processors/khalaProcessor.js",
-    "process:polkadot": "node lib/processors/polkadotProcessor.js",
-    "process:kusama": "node lib/processors/kusamaProcessor.js",
+    "dev:khala": "NETWORK=khala node -r dotenv/config lib/processor.js",
+    "dev:polkadot": "NETWORK=polkadot node -r dotenv/config lib/processor.js",
+    "dev:kusama": "NETWORK=kusama node -r dotenv/config lib/processor.js",
```

Comment:
On Balances we did something different:
https://github.com/litentry/squid/blob/feature/firesquid/prawns/balances/package.json#L9

Not related to this PR but maybe the firesquid branch where we should definitively align all these `package.json` or find a way to move these commands to the top level.

Reply:
I personally prefer this approach as it allows the developer to swap between networks without changing the env file.

Although I guess you could just do `NETWORK=khala yarn dev` I guess that works?

Reply:
Yeah, I prefer this approach. It was just about aligning the commands we do on each prawn. But not related exclusively to this PR, just a thought on the project.